geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -103,8 +103,13 @@ object Transformers extends JavaTokenParsers {
   }
 
   class EvaluationContext(fieldNameMap: Map[String, Int], var computedFields: Array[Any]) {
```

Comment:
I think global params should be appended to the end of computedFields rather than used as a lookup Map here.